Gems/ScriptCanvas/Code/Editor/Assets/ScriptCanvasFileHandling.cpp

```diff
@@ -269,6 +269,10 @@ namespace ScriptCanvas
         const auto& asString = fileStringOutcome.GetValue();
         result.m_deserializeResult = Deserialize(asString, makeEntityIdsUnique, loadReferencedAssets);
         result.m_isSuccess = result.m_deserializeResult;
+        if (!result.m_deserializeResult.m_isSuccessful)
+        {
+            result.m_fileReadErrors = "Script Canvas Graph Deserialization Failed - " + result.m_deserializeResult.m_errors + "\n";
```

Comment:
The "Script Canvas Graph Deserialization Failed - " message seems like a redundant message that can be skipped altogether. The error message in m_errors is understandable enough imo.

Reply:
I prefer to keep the message since it is kind of confusing that only use some of the error messages from the `Deserialize` function directly without mentioning the errors are related to deserialization.

Reply:
But it's already mentioned that errors are due to deserialization 
```
JSON (and failsafe XML) deserialize attempt failed
```
Isn't that enough?

Reply:
There are other error messages in `Deserialize` like:

- "Failed to find graph data after loading source"
- "Entity loaded in bad state"
- ...